src/mds/MDLog.h

```diff
@@ -65,30 +64,32 @@ class MDSRank;
 class LogSegment;
 class ESubtreeMap;
 
+using LogSegmentRef = boost::intrusive_ptr<LogSegment>;
```

Comment:
Ugh.. where are `intrusive_ptr_add_ref` and `intrusive_ptr_release` implemented? o_O

Reply:
Oh I see, LogSegment is a subclass of RefCountedObject. Interesting.